packages/docker/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@nx/docker",
+  "description": "The Nx Plugin for Docker to aid in containerizing projects.",
+  "version": "0.0.1",
+  "type": "commonjs",
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/nrwl/nx.git",
+    "directory": "packages/docker"
+  },
+  "bugs": {
+    "url": "https://github.com/nrwl/nx/issues"
+  },
+  "keywords": [
+    "Monorepo",
+    "Docker",
+    "Containers"
+  ],
+  "author": "Colum Ferry",
```

Comment:
We do not usually attribute personal authorship, although I see you are the only other person who has done so in the repo...